lib/config-generator.js

```diff
@@ -185,19 +181,6 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
             }
         }
 
-        if (this.answers.env?.length > 0) {
-            this.result.devDependencies.push("globals");
-            importContent += "import globals from \"globals\";\n";
-            const envContent = {
-                browser: "globals: globals.browser",
-                node: "globals: globals.node",
-                "browser,node": "globals: {...globals.browser, ...globals.node}"
-            };
-
-            exportContent += `  {languageOptions: { ${envContent[this.answers.env.join(",")]} }},\n`;
-        }
-
-
```

Comment:
Move these 2 items to the end - to avoid being overwritten by 3rd-party configs.